web/marketing/css/style.css

```diff
@@ -178,4 +178,8 @@ input{
 .edit-offer:hover{
     text-decoration: underline;
     color: #656565;
+}
+
+.stop-offer:hover {
```

Comment:
the `:hover` part is not necessary here, you can default it to `cursor:pointer`.